src/main/java/net/pms/dlna/DVDISOFile.java

```diff
@@ -90,7 +109,7 @@ protected void resolveOnce() {
 			"null",
 			"-dvd-device",
 			ProcessUtil.getShortFileNameIfWideChars(file.getAbsolutePath()),
-			"dvd://1"
+			"dvd://"
```

Comment:
@Sami32 What are you reacting to? The number is the "track number", here we're parsing the disc itself - thus no track. That said, it works either way.